src/ChainSafe.Gaming.Marketplace/MarketplaceClient.cs

```diff
@@ -64,39 +63,21 @@ public MarketplaceClient(IHttpClient httpClient, IProjectConfig projectConfig, I
         }
 
         /// <summary>
-        /// Loads first marketplace page or a next one.
+        /// Loads marketplace page via previous page.
         /// </summary>
-        /// <param name="currentPage">Last loaded page.</param>
+        /// <param name="prevPage">Previous page.</param>
         /// <returns>A <see cref="Task"/> representing the asynchronous operation.</returns>
-        public Task<MarketplacePage> LoadPage(
-            MarketplacePage? currentPage = null,
-            string? marketplaceId = null,
-            string? filterTokenContract = null,
-            MarketplaceItemStatus? filterStatus = null,
-            MarketplaceSortType sortType = MarketplaceSortType.None,
-            MarketplaceSortOrder? sortOrder = null)
+        public Task<MarketplacePage> LoadPage(MarketplacePage? prevPage = null)
```

Comment:
This looks like it overwrites the recent marketplace update. Please make sure this doesn't get written to dev.

Reply:
ok